Assets/Scripts/Game/EnemyMotor.cs

```diff
@@ -309,22 +325,32 @@ void KnockbackMovement()
         void ApplyGravity()
         {
             // Apply gravity
-            if (!flies && !swims && !IsLevitating && !controller.isGrounded)
+            if (entity.IsSlowFalling && !flies && !swims && !controller.isGrounded)
+            {
+                Vector3 velocity = controller.velocity * 0.97f; //gradually slow x/z movement
```

Comment:
Where does this code come from? I thought this was moved from somewhere else, but I can't find it. Is this some mod code you accidentally slipped here, or an intended feature of the PR?

Reply:
It's an intended feature of the PR.
There wasn't any logic in ApplyGravity to handle enemy slowfalling so I added it in.

Reply:
I think I copied the logic from the player motor.

Reply:
I feel like your implementation is a bit too specific for general DFU code. Mostly the part where it slows horizontal movement by 3% every frame. The fall speed is probably debatable too.

I get it's probably a bit hard to add through a mod otherwise... but it could be an event at this point.

I do think it's fine for the "lastGroundedY" to be updated every frame while slow falling otherwise, that should be how it works in all cases.

Reply:
DFU does not currently allow enemies to slowfall, only fly.
In the base game, enemies don't have slowfall in their spellbooks, so it doesn't factor in.
If a mod (like mine) applies a slowfall effect to an enemy or adds it to their spellbook, it would be completely ineffective.